src/Nri/Ui/Confetti/V2.elm

```diff
@@ -0,0 +1,197 @@
+module Nri.Ui.Confetti.V2 exposing
+    ( Model, init
+    , view
+    , Msg, burst, update, updatePageWidth, subscriptions
+    )
+
+{-|
+
+@docs Model, init
+@docs view
+@docs Msg, burst, update, updatePageWidth, subscriptions
+
+Changes from V1:
+
+  - removes custom words from confetti
+
+-}
+
+import Css exposing (Color)
+import Html.Styled as Html
+import Html.Styled.Attributes as Attributes exposing (css)
+import Nri.Ui.Colors.V1 as Colors
+import Particle exposing (Particle)
+import Particle.System as ParticleSystem
+import Random exposing (Generator)
+import Random.Float exposing (normal)
+
+
+
+-- MODEL
+
+
+{-| -}
+type Model
+    = System (ParticleSystem.System Confetti) Float
+
+
+type Confetti
+    = Square
+        { color : Color
+        , rotations : Float
+        , rotationOffset : Float
+        }
+
+
+{-| -}
+init : Float -> Model
+init center =
+    System
+        (ParticleSystem.init (Random.initialSeed 0))
+        center
+
+
+
+-- VIEW
+
+
+{-| -}
+view : Model -> Html.Html msg
+view (System system _) =
+    system
+        |> ParticleSystem.viewCustom viewConfetti
+            (Html.div
+                [ Attributes.style "position" "absolute"
+                , Attributes.style "top" "0"
+                , Attributes.style "left" "0"
+                , Attributes.style "width" "100%"
+                , Attributes.style "height" "100vh"
+                , Attributes.style "pointer-events" "none"
+                ]
+            )
+
+
+viewConfetti : Particle Confetti -> Html.Html msg
+viewConfetti particle =
+    let
+        lifetime =
+            Particle.lifetimePercent particle
+    in
+    case Particle.data particle of
+        Square { color, rotationOffset, rotations } ->
+            Html.div
```

Comment:
these were only HTML because we wanted to be able to style them with CSS and add text. Now that they're just squares, would SVG be more performant?